### ISSUE TITLE
GCP: Control the default allow-external-access FW rules

### ISSUE THREAD

**Issue by georgi-lozev** [gardener/gardener-extensions#345]
We have Gardener shoot clusters running in GCP with firewall rules like the following.

```
shoot-prod-allow-external-access
Ingress
Apply to all    
IP ranges: 0.0.0.0/0
tcp:80,443
Allow
1000
```

The fact that the port 80 is open to the internet is being classified as a high security risk.
There is nothing from our services actually listening on that port and it will be good if we could close it. 

Is there a way for you to allow us to have control over that configuration and open it only if needed?

**Comment by georgi-lozev** [comment]
Here is the suspected code.
https://github.com/gardener/gardener-extensions/blob/464effceb7d28820fe819b236ac90336a9ad85d9/controllers/provider-gcp/charts/internal/gcp-infra/templates/main.tf#L71-L80

**Comment by ialidzhikov** [comment]
It looks like we have 2 similar issues for appr. the same thing - this one and gardener/machine-controller-manager#339.

**Comment by ialidzhikov** [comment]
@zanetworker , @rfranzke can we close this one as the gcp worker nodes have no longer external IP?

**Comment by rfranzke** [comment]
We will have these firewall rules. I think the issue is more about these rules, isn't it? Can we get rid of them? cc @DockToFuture

**Comment by ialidzhikov** [comment]
I guess the concern here was that port 80 was exposed but as we don't have now external IPs it should no longer be a security issue, right?

**Comment by zanetworker** [comment]
it should be no longer a security issue, but these rules won't be used anywhere, and would maybe give the wrong impression that we expose something.

**Comment by rfranzke** [comment]
Exactly like @zanetworker wrote. Hence, can we remove them?

**Comment by ialidzhikov** [comment]
g/gardenctl currently use it - ref https://github.com/gardener/gardenctl/blob/fea57bfded46b528b89f1635ce6fb9dc27a50892/pkg/cmd/ssh_gcp.go#L103-L114

**Comment by rfranzke** [comment]
ping @zanetworker @DockToFuture

**Comment by zanetworker** [comment]
@rfranzke yes we can remove them, however, I am not sure if clients such as gardenctl is still using them.
